pandas/core/indexes/datetimes.py

```diff
@@ -2091,7 +2094,7 @@ def bdate_range(start=None, end=None, periods=None, freq='B', tz=None,
 def cdate_range(start=None, end=None, periods=None, freq='C', tz=None,
                 normalize=True, name=None, closed=None, **kwargs):
     """
-    **EXPERIMENTAL** Return a fixed frequency datetime index, with
+    **EXPERIMENTAL** Return a fixed frequency DatetimeIndex, with
```

Comment:
side issue we can prob remove the experimental (but new PR for that)

Reply:
Did this in #17554, as there were other doc related fixes that needed to be made.